src/store/assets/getters.ts

```diff
@@ -52,6 +52,10 @@ const getters = defineGetters<AssetsState>()({
     const { getters } = assetsGetterContext(args);
     return getters.assetDataByAddress(XOR.address);
   },
+  eth(...args): Nullable<RegisteredAccountAsset> {
```

Comment:
If it's used only on one page, maybe better to define it in this page component?